tutorial/cli.md

```diff
@@ -211,37 +212,38 @@ func NewModuleClient(storeKey string, cdc *amino.Codec) ModuleClient {
 
 // GetQueryCmd returns the cli query commands for this module
 func (mc ModuleClient) GetQueryCmd() *cobra.Command {
-	// Group gov queries under a subcommand
-	govQueryCmd := &cobra.Command{
+	// Group nameservice queries under a subcommand
```

Comment:
Thank you for catching my copy pasta here 🙈

Reply:
If you are making changes in the code in the `.md` files, please also make the same change in the corresponding `.go` file!